js/main.js

```diff
@@ -1,382 +0,0 @@
-  addModalCloseButton("aboutHeaderBtn");
-  createElement("aboutContent", "div", ["modal-body"], "aboutBody", "Roger's Weather App\nAwesome Inc. Web Developer Bootcamp\nFall 2022\nMIT License");
-  createElement("aboutContent", "div", ["modal-footer"], "aboutFooter");
-  createElement("aboutFooter", "button", ["btn", "btn-secondary"], "aboutFooterCloseButton", "Dismiss");
-  addModalCloseButton("aboutFooterCloseButton");
-
-  // Contact Modal
-  createElement("appContainer", "div", ["modal", "fade"], "contactModal");
-  createElement("contactModal", "div", ["modal-dialog"], "contactDialog");
-  createElement("contactDialog", "div", ["modal-content"], "contactContent");
-  createElement("contactContent", "div", ["modal-header"], "contactHeader");
-  createElement("contactHeader", "h5", ["modal-title"], "contactTitle", "Contact the Developer");
-  createElement("contactHeader", "button", ["btn-close"], "contactHeaderBtn");
-  addModalCloseButton("contactHeaderBtn");
-  createElement("contactContent", "div", ["modal-body"], "contactBody", "Roger Mullins\nrogermullins.mba@gmail.com");
-  createElement("contactContent", "div", ["modal-footer"], "contactFooter");
-  createElement("contactFooter", "button", ["btn", "btn-secondary"], "contactFooterCloseButton", "Dismiss");
-  addModalCloseButton("contactFooterCloseButton");
-};
-
-function getZip() {
-  zipIn = document.getElementById("zip-in").value;
-
-  if (zipValidates(zipIn)) {
-    callAPI(zipIn);
-  } else {
-    alert("Invalid ZIP code.");
-  }
-
-  // Reset Input Box
-  zipIn.value = "";
-}
-
-function getTime() {
-  // Reused from alarm clock project -RM
-  let timeHours = 0;
-  let timeMinutes = 0;
-  let timeSeconds = 0;
-  let alarmHours = 0;
-  let alarmMinutes = 0;
-  let alarmSeconds = 0;
-  
-
-  let today = new Date();
-  let time = today.getHours() + ":";
-  if (today.getMinutes() < 10) {
-      time = time + "0";
-  }
-  time = time + today.getMinutes() + ":";
-  if (today.getSeconds() < 10) {
-      time = time + "0";
-  }
-  time = time + today.getSeconds();
-  let date = (today.getMonth()+1)+'-'+today.getDate() +'-'+ today.getFullYear();
-
-  let timeBox = document.getElementById("showTime");
-  let dateBox = document.getElementById("showDate");
-  timeBox.innerText = time;
-  dateBox.innerText = date;
-}
-
-function capitalizeConditions(stringIn) {
-  // Takes string of current conditions returned by API (all lower case)
-  // Returns same string with first letter capitalized
-  const firstLetter = stringIn.charAt(0);
-  const capitalized = firstLetter.toUpperCase();
-  const restOfString = stringIn.slice(1);
-  const stringOut = capitalized + restOfString;
-  return stringOut;
-}
-
-async function callAPI(zipIn) {
-  // Builds API call string and gets data from API
-  const apiKey = "a6063ef55e23ba5afdf671e31f5cdf4b";
-  const apiCall = ("https://api.openweathermap.org/data/2.5/weather?zip=" + zipIn + ",us&appid=" + apiKey);
```

Comment:
These are better served as constants at the top of the file (or eventually in their own file). 

Additionally we want to get away from building up the full URL in a concatenated string and use Axios a little better. 

```javascript 
const API_KEY = "a6063ef55e23ba5afdf671e31f5cdf4b";
const BASE_URL = "https://api.openweathermap.org/data/2.5/weather"

let options = {
  params: {
    appid: API_KEY,
    zip: zipIn,
  }
}

axios.get(BASE_URL, options); // concatenates the params on the end dynamically